front/assets/style/modals.scss

```diff
@@ -1,6 +1,28 @@
+@media (max-width: 959px) {
```

Comment:
use breakpoints instead of hard coding, actually we need to replace breakpoints in your scss with vuetify ones, i created ticket for this but still would be easier to replace them if we use rather than if we have this value hardcoded
https://github.com/ezl/dailyidea.com/issues/415

Reply:
Ok, added variables